### DISCUSSION TITLE
iCloud storage

### DISCUSSION THREAD

**Discussion by Drallas** [lra/mackup#1903] [Q&A] [answered]
Where are the dotfiles stored if I use:

```bash
[storage]
engine = icloud
```
Currently, I use:

```bash
[storage]
engine = file_system
path = Documents/Backups/Dotfiles/
directory = Mackup
```

This folder is already located in my iCloud.
So I thought perhaps is there a more efficient way, using the iCloud driver?

PS: This part is missing in the documentation!

**Comment by joshmedeski** [comment, marked as answer]
Hey @Drallas,

icloud is a supported solution.

[Check out the docs](https://github.com/lra/mackup/blob/master/doc/README.md#icloud).

It looks for a mackup folder in this location: `~/Library/Mobile Documents/com~apple~CloudDocs/`

If your files aren't there you'll have to continue using the current config. Hope that helps.

**Comment by Drallas** [comment]
Ok thanks.

When iCloud is enabled, a Mackup folder will be expected at `~/Library/Mobile\ Documents/com\~apple\~CloudDocs/`and used. 

Seems more difficult to find the folder there; compared to where I keep it now. ~/Documents/Backups/Dotfiles/.

**Reply by joshmedeski** [reply]
I agree, I use ~/repos/dotfiles myself and use GitHub to manage changes.
